libs/application/templates/new-primary-school/src/fields/RelativesTableRepeater/index.tsx

```diff
@@ -51,12 +51,10 @@ const RelativesTableRepeater: FC<React.PropsWithChildren<FieldBaseProps>> = ({
         maxRows: 4,
         editField: true,
         fields: {
-          fullName: {
-            component: 'input',
-            label: newPrimarySchoolMessages.shared.fullName,
-            width: 'half',
-            type: 'text',
-            dataTestId: 'relative-full-name',
+          NationalIdWithName: {
```

Comment:
Change to lowercase? 🤔 

```suggestion
          nationalIdWithName: {
```